src/pages/starter-kit/checkout/connect.md

```diff
@@ -18,17 +18,15 @@ To use the Adobe Commerce HTTP Client, update `COMMERCE_BASE_URL=<commerce_insta
 
 Depending on your Adobe Commerce setup, there are two options to authenticate and communicate with App Builder:
 
-- [Configure Adobe Identity Management Service (IMS)](#configure-adobe-identity-management-service-ims)
-
 - [Configure Commerce Integration](#create-a-commerce-integration)
 
-If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, than client instantiation will fail.
+- [Configure Adobe Identity Management Service (IMS)](#app-builder-authentication-ims)
 
-### Configure Adobe Identity Management Service (IMS)
+If a Commerce integration is detected, it has precedence over IMS authentication. However, if neither option is detected or configured, then client instantiation will fail.
 
-IMS authentication requires the following setup:
+### App Builder authentication (IMS)
```

Comment:
It's ok to keep the name as `Adobe Identity Management Service (IMS)` (since both methods are for App Builder authentication):
```
App Builder authentication (IMS) => Adobe Identity Management Service (IMS)
```

Also, the order of the bullet items is reversed, but we can keep it as before so that the sections are presented in order:
```
Depending on your Adobe Commerce setup, there are two options to authenticate and communicate with App Builder:
- Configure Adobe Identity Management Service (IMS) ==> first
- Configure Commerce Integration ==> second
```